collection-scripts/gather_run

```diff
@@ -64,3 +64,10 @@ wait_bg
 
 # Create rotated log symlinks after everything else has finished
 rotated_logs_symlinks
+
+# create an easy to download tar from the whole content
+tar \
+    --exclude='must-gather.tar.xz' \
+    --warning=no-file-changed --ignore-failed  \
+    -cJf \
+    "${BASE_COLLECTION_PATH}/must-gather.tar.xz" "${BASE_COLLECTION_PATH}" || true
```

Comment:
I'm happy to eventually push follow ups.

Reply:
Hi,
I'm sorry because I'm terribly late here.
I think it's a good idea and I'm in line with @Akrog 's comment, especially because we can really benefit from this change in CI and it's very useful for customers as well.
I think it's a nice feature and I'm +1 to this, so feel free to land this change, or, if you have cycles, to add more patches in case to improve the current code.
It would also be nice to mention about this in the README, and, in case of a follow up,  add the new variable required to customize the resulting location.
I would also have a +1 by someone from the CI team, to make sure they're aware and can follow up in case something is needed from a `ci-framework` perspective.
In general +1 to land this change.

Reply:
@fmount no worries. As it is visible here I also had no time to get back to this, even though I agree with the improvement ideas.

> I would also have a +1 by someone from the CI team, to make sure they're aware and can follow up in case something is needed from a ci-framework perspective.

I used the default location within the same directory everything is collected to sidestep the need of changing the ci framework's log gathering code. :)

Reply:
Finally had time to get back to this. I added the requested configuration improvements.